lib/dotnet-parser.js

```diff
@@ -64,6 +64,9 @@ module.exports = {
       version: '0.0.0',
       packageFormatVersion: 'nuget:0.0.0',
       dependencies: {},
+      meta: {
+        targetFramework: dotnetVersions[0].original, //TODO implement for more than one TF
```

Comment:
What happens when a user already has the project monitored and will now start sending back targetRuntime? Please run through this scenario and confirm?

Reply:
It will create a new project. Which is the expected behavior since we create a new project for each TF

Reply:
Would we want to run a migration for all currently monitored projects from CLI to add the first TF from their csproj?

Reply:
we need to discuss this, as it may be completely surprising to a user that instead of keeping history of tests we started a new project